crypto/cpu-intel.c

```diff
@@ -62,11 +62,15 @@
 #include <inttypes.h>
 
 #if defined(OPENSSL_WINDOWS)
+#ifdef _MSC_VER
```

Comment:
NIT: we always use `#if defined(...)` in the C code. This is a convention from BoringSSL.

I'll fix this for you.